### DISCUSSION TITLE
Incorrect icons are shown

### DISCUSSION THREAD

**Discussion by ozzy1136** [amnweb/yasb#6] [Q&A] [answered]
![Screenshot 2024-07-29 132408](https://github.com/user-attachments/assets/6586161d-7c56-49ae-96e7-62fda3a688f2)

I cloned the repo, but the icons seem to not show correctly e.g. battery, shutdown. I didn't edit the codes used for the icons; the only changes made to the config file were for the weather location and the clock text.

Is the user supposed to edit the unicode codes or could there be a Windows setting that I need to change?

**Comment by amnweb** [comment, marked as answer]
Install Nerd Font `JetBrainsMono NFP` for me, is the best one.

**Comment by ozzy1136** [comment]
Of course, I don't know why I didn't realize that I needed a font to display those characters. Thanks for that tip!
This is a great project btw
